src/Ormolu/Parser/CommentStream.hs

```diff
@@ -163,7 +166,22 @@ mkComment ls (L l s) = (ls', comment)
             "{-" -> (False, ls'')
             _ -> (True, ls'')
     dropTrailing = L.dropWhileEnd isSpace
-    startIndent = srcSpanStartCol l - 1
+    startIndent
+      -- multiline comments have an "absolute" start at column 0
```

Comment:
I don't think this comment is correct. What happens if that if a multiline comment sits at column 6 we actually drop all 6 spaces (unless there are other lines with fewer number of spaces in the indent) and this becomes the output of `mkComment`. I think we subtract 1 to account for the fact that `SrcSpan` counts columns from 1. Spaces are indeed restored but later by the printing engine, based on the actual indent the comment has when printing.

Reply:
Yeah you're right, I misunderstood what was happening there. I will correct it :)
I suppose that the next comment (relative to the single-line comment case) is correct though?

Reply:
Yes, the second one is correct.